README.md

```diff
@@ -8,7 +8,8 @@
 
 &nbsp;
 
-[![CI Tests](https://github.com/iasql/iasql/actions/workflows/ci.yml/badge.svg)](https://github.com/iasql/iasql/actions/workflows/ci.yml)
+[![CI Tests](https://github.com/iasql/iasql/actions/workflows/tests.yml/badge.svg)](https://github.com/iasql/iasql/actions/workflows/tests.yml)
```

Comment:
I wonder if there's a more specific name for this than `CI Tests` now that we have `Dashboard Tests`

Reply:
I was wondering if called it `Integration Tests`

Reply:
```suggestion
[![Integration Tests](https://github.com/iasql/iasql/actions/workflows/tests.yml/badge.svg)](https://github.com/iasql/iasql/actions/workflows/tests.yml)
```